<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PdfViewWidget</class>
 <widget class="QWidget" name="PdfViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ZoomableGraphicsView" name="pdf_graphics_view">
     <property name="renderHints">
      <set>QPainter::RenderHint::Antialiasing|QPainter::RenderHint::SmoothPixmapTransform|QPainter::RenderHint::TextAntialiasing</set>
     </property>
     <property name="dragMode">
      <enum>QGraphicsView::DragMode::NoDrag</enum>
     </property>
     <property name="resizeAnchor">
      <enum>QGraphicsView::ViewportAnchor::AnchorUnderMouse</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZoomableGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>core.widgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
